src/lobsterpy/plotting/__init__.py

```diff
@@ -1234,3 +1235,128 @@ def get_plot(
                 ax.scatter(x, y, s=marker_size, marker=marker_style, label=label)
 
         return plt
+
+
+class BWDFPlotter:
+    """
+    Plotter to generate Bond weighted distribution functions using ICOHP or ICOBI or ICOOP as weights.
+
+    :param are_coops: Bool indicating that populations are ICOOPs, not ICOHPs.
+                    Defaults to False for COHPs.
```

Comment:
Maybe you could update the doc of these params here and in other plotting classes: imo the `Defaults to False for COHPs.` is not very intuitive and could be replaced with something like `Defaults to False, i.e., COHPs are plotted.`

Reply:
Done